### PR TITLE
Update Docker development env to official PHP image.

### PR DESCRIPTION
**Pull request by newPOPE**
I updated docker dev environment a little. 

- switched to official image (PHP5.4)
- simplify Dockerfile
- drop entrypoint (it is other thing/barrier for other devs)

### CODE REVIEW

**Comment by newPOPE** [comment]
ping @lorenzo

**Comment by garex** [comment]
php5.4 is not compatible with latest phpunit used in this project. So better to switch to 5.6 or even to 7.0

**Comment by lorenzo** [comment]
I have no idea how I forgot to merge this. Any improvements, please open another PR

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,13 +66,8 @@ If you do not wish to install MySQL and Postgres locally to run unit tests
...
 both database, install all development dependencies and run all unit tests.
 
-To get started, just run `docker-compose run phinx`. It will download all 
-images, install all dependencies, hook them up and run all unit tests
-(note: SQLServer tests are not supported, yet).
-
-You can also use `docker-compose run phinx install` to update/reinstall
-php composer packages manually. If you wish to drop into a shell, just run
-`docker-compose run phinx bash`, which will drop you right into a shell.
+To get started, just run `dco run --rm phinx`. It will download all
```
Comment (glensc) [inline CONTRIBUTING.md]: `dco`?

it's not standard alias, wrapper installed by docker-compose

```

[~] ➔ docker-compose --version
docker-compose version 1.21.0, build 5920eb0
[~] ➔ dco --version
-bash: dco: command not found
```

Comment (newPOPE) [reply CONTRIBUTING.md]: You are right. `dco` is helper installed via bash-it and it is not standard.

**Pull request merged**